database.go

```diff
@@ -1049,6 +1049,35 @@ func (rp *RetentionPolicy) shardGroupByID(shardID uint64) *ShardGroup {
 	return nil
 }
 
+// dropMeasurement will remove a measurement from the index.
+func (db *database) dropMeasurement(name string) error {
```

Comment:
I was expecting this to be different. To drop a measurement, you could just drop an entire bucket in Bolt. I don't think that is what is happening here. Perhaps it doesn't matter.